### DISCUSSION TITLE
lastfm problem

### DISCUSSION THREAD

**Discussion by Sacrylegio** [pear-devs/pear-desktop#1241] [Q&A] [answered]
I don't understand why the plays aren't showing on lastfm, I connected correctly to the YouTube music site, and I activated the function on the app, I tried waiting weeks but it doesn't seem to be working.
could someone help me? Thanks in advance

**Comment by JellyBrick** [comment, marked as answer]
fixed at https://github.com/th-ch/youtube-music/commit/cc0bfae0676e63a683f91f336082fdaa59b7af31
